server/lib/scene/scene.dailyUpdate.js

```diff
@@ -0,0 +1,41 @@
+const schedule = require('node-schedule');
+const { EVENTS } = require('../../utils/constants');
+
+/**
+ * @description Add Schedule job for sunset and sunrise for each house
+ * @returns {Promise} Resolve when success.
+ * @example
+ * scene.dailyUpdate();
+ */
+async function dailyUpdate() {
+  const houses = await this.house.get();
+
+  const jobs = [];
+
+  houses.forEach((house) => {
+    const times = this.sunCalc.getTimes(new Date(), house.latitude, house.longitude);
```

Comment:
If the house has no location defined (it's possible), it'll crash.

Reply:
Fixed an test added